### PR TITLE
9970 Story: Generate 30 Day Notice of Trial

### PR DESCRIPTION
**Pull request by zachrog**
This PR includes only the functionality needed to implement this story, it does not include any of the refactoring we discussed with Chris and Tenille because it requires a migration and currently migration deploys are breaking smoke tests (nothing is actually broken other than the test itself). The refactoring can be merged separately once that issue has been addressed.

### CODE REVIEW

**Review by rachelschneiderman — APPROVED** [review]

File: shared/src/business/entities/cases/Case.removeFromHearing.test.ts
Comment (jimlerza) [inline shared/src/business/entities/cases/Case.removeFromHearing.test.ts:1]: Why this change?

(Several other files have similar changes, too)

Comment (codyseibert) [reply shared/src/business/entities/cases/Case.removeFromHearing.test.ts:1]: I converted them back to esm imports

**Review by TomElliottFlexion — APPROVED** [review]

File: setup-local-env.sh [hunk 1]
```diff
@@ -17,3 +17,4 @@ export EFCMS_DOMAIN=localhost
 export ELASTICSEARCH_HOST=localhost
 export ELASTICSEARCH_ENDPOINT=http://localhost:9200
 export DYNAMODB_ENDPOINT=http://localhost:8000
+export IRS_SUPERUSER_EMAIL=irs-superuser@example.com
```
Comment (mmarcotte) [inline setup-local-env.sh]: can you add a newline please?

**Review by mmarcotte — APPROVED** [review]
